tests/fuzz/fuzz_data_producer.c

```diff
@@ -28,12 +28,12 @@ void FUZZ_dataProducer_free(FUZZ_dataProducer_t *producer) { free(producer); }
 
 uint32_t FUZZ_dataProducer_uint32Range(FUZZ_dataProducer_t *producer, uint32_t min,
                                   uint32_t max) {
-    FUZZ_ASSERT(min <= max);
```

Comment:
This change actually reduces readability of the code. It is more readable to assert `min <= max` before using that property for `range = max - min`.

The only benefit of this change is that we get closer to complying with C89 rules... but why?